lab04/sections/results.tex

```diff
@@ -1,3 +1,32 @@
+\setkeys{Gin}{width=0.9\textwidth}
 \section{Results}\label{sec:results}
-Describe the apparatus and measurement technique(s).
-Present the data.
+Four circuits were assembled and analyzed using a function generator to apply a stimulus, and an oscilloscope to measure the output of the system over the input. A detailed diagram of the circuits which produced these outputs can be referenced in the lab manual \cite{lab-manual}.
```

Comment:
Schematics are included in `graphics/schematic-XXX`. You'll need to include them with the resistance values to make claims about what the expected gain should be.